roles/kafka_broker/tasks/kraft_tasks.yml

```
@@ -0,0 +1,17 @@
+---
+- name: extract uuid from meta.properties
```

```suggestion
- name: Extract ClusterId from meta.properties on KRaft Controller
```

Tasks should be named similarly to others, w capital letters

---

Will take care of this and update all task names